mithril-aggregator/src/database/provider/cardano_transaction.rs

```diff
@@ -315,13 +336,14 @@ mod tests {
             .unwrap()
     }
 
+    // TODO Is it really a useful test ?
```

Comment:
Same here, comment is not clear. Could you be more specific?

Same remark for all the below TODOs.